src/components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { useSideBarContext } from "../../utils/contexts/SideBarContext";
+import "./SideBar.styles.css";
+
+function SideBar({ children }) {
+  const { sidebar, setSidebar } = useSideBarContext();
+
+  const options = [
+    { text: "Notes", link: "/home", icon: "/icons/bulb.png" },
+    { text: "Archive", link: "/archive", icon: "/icons/archive.png" },
+  ];
```

Comment:
same as before, this variable could be defined as a constant outside the component definition